ui/docs/tutorial/debug-task.md

```diff
@@ -39,7 +39,7 @@ VNC connection to a running task execution environment.
 
 There are two ways to access this mode. The easiest is to enter the task's ID
 in the [task inspector](https://tools.taskcluster.net/groups). Under
-"actions", select "create interactive task". If you have permission to create
+"actions", select "Create with SSH/VNC task". If you have permission to create
```

Comment:
This pull-request seems to be out of an older commit. You will want to rebase on master to remove these changes from this pr.

```diff
git pull upstream master --rebase
git push origin Secrets_view_UX_581 -f
```